.travis.yml

```diff
@@ -93,6 +100,7 @@ before_script:
   - make -j2
 
 script:
+  - $CPPCHECK
```

Comment:
Does `cppcheck` generate any output if it passes? Right now, the Travis logs just show `$CPPCHECK`, so we don't know if the environment variable was empty of if it did anything at all.

Reply:
no output = good
otherwise all problems are listed separately

Reply:
My worry is that no output could also mean "nothing happened," e.g. if we misconfigure and `$CPPCHECK` is not properly set.